spec/requests/dois_spec.rb

```diff
@@ -3296,45 +3354,47 @@
       end
     end
 
-    context 'update geoLocationPoint' do
-      let(:geo_locations) { [
-        {
-          "geoLocationPoint" => {
-            "pointLatitude" => "49.0850736",
-            "pointLongitude" => "-123.3300992"
-          }
-        }] }
-      let(:update_attributes) do
-        {
-          "data" => {
-            "type" => "dois",
-            "attributes" => {
-              "geoLocations" => geo_locations
-            }
-          }
-        }
-      end
+    # TODO needs updating
```

Comment:
Is this broken due to the identifiers change? I can't quite see why this test would fail.

Reply:
I don't see a connection to changes in this pull request, that is why I commented it out. Need to look at it independently.